rules-java-ee/addon/src/main/java/org/jboss/windup/rules/apps/javaee/model/HasApplications.java

```diff
@@ -19,29 +19,10 @@
 
     @Override
     @JavaHandler
-    boolean belongsToProject(ProjectModel projectModel);
-
-    @Override
-    @JavaHandler
     Iterable<ProjectModel> getRootProjectModels();
 
     abstract class Impl implements HasApplications, JavaHandlerContext<Vertex>, BelongsToProject
     {
-        @Override
-        public boolean belongsToProject(ProjectModel project)
```

Comment:
This change is unrelated, but I removed it as the operation was unused and inherently unsafe. It doesn't deal with shared libraries correctly.